dependency/retrieval/components/output_test.go

```diff
@@ -0,0 +1,76 @@
+package components_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/paketo-buildpacks/tini/dependency/retrieval/components"
+	"github.com/paketo-buildpacks/packit/v2/cargo"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testOutput(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect    = NewWithT(t).Expect
+		outputDir string
+		depDate   time.Time
+	)
+
+	it.Before(func() {
+		var err error
+		outputDir, err = os.MkdirTemp("", "")
+		Expect(err).NotTo(HaveOccurred())
+
+		depDate, err = time.ParseInLocation("2006-01-02", "2024-11-12", time.UTC)
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	it.After(func() {
+		Expect(os.RemoveAll(outputDir)).To(Succeed())
+	})
+
+	context("WriteOutput", func() {
+		it("will write an output file", func() {
+			err := components.WriteOutput(filepath.Join(outputDir, "output.json"), []cargo.ConfigMetadataDependency{
+				{
+					DeprecationDate: &depDate,
+					Licenses:        []interface{}{"MIT", "MIT-0"},
+					Name:            ".NET Core SDK",
```

Comment:
was this (using .NET Core SDK as an example) intentional? I can see where it might not matter but just making sure.

Reply:
Oh yeah. I am not really sure that is matters. I pulled the functionality over from another repo because it is relatively generic